src/components/DatePicker/styles/Wrapper.js

```diff
@@ -1,4 +1,5 @@
 import styled, { css, keyframes } from 'styled-components';
+import get from 'extensions/themeGet';
```

Comment:
As I can see from the code, we mixed the `${get()}` and `var` here, I don't like that idea and think that we should choose one of the approaches. I'd prefer `var`